app/views/deckster/deck/_card.html.erb

```diff
@@ -14,16 +15,28 @@
         <li>
           <a title="Popout" class='deckster-control deckster-popout-handle' style="display: none"><i class="fa fa-share"></i></a>
         </li>
+        <% if expandable %>
         <li>
           <a title='Collapse' class='deckster-control deckster-collapse-handle deckster-expand-collapse-handle' style="display: none"><i class="fa fa-compress"></i></a>
         </li>
         <li>
           <a title='Expand' class='deckster-control deckster-expand-handle deckster-expand-collapse-handle'><i class="fa fa-expand"></i></a>
         </li>
+        <% end %>
+        <li>
+          <a title="Hide" class='deckster-control deckster-hide-handle'><i class="fa fa-close"></i></a>
+        </li>
       </ul>
     </div>
   </nav>
   <div class="deckster-content">
+    <% if !visualizations.nil? %>
```

Comment:
we should move the card data out to partials and make one for summary 
/app/views/deckster/cards/ 

there should already be an example partial there for the icons card.. this will allow us to keep with the design specs for pre built cards @lu-c & @hochardga